lib/processor/deps.edn

```diff
@@ -9,29 +9,29 @@
 
         garden/garden {:mvn/version "1.3.10"}
         ;girouette/girouette {:local/root "../girouette"}
-        girouette/girouette {:mvn/version "0.0.2"}}
+        girouette/girouette {:mvn/version "0.0.3"}}
```

Comment:
It works without changing the version. The version used in the user's project will override it.